docs/introduction.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Introduction
+id: introduction
+---
+
+# What is `granary`?
+
+Granary is a job runner for cloud-based geospatial machine learning.
+Its goal is to simplify running models and to track and serve
+the results of predictions. It puts a REST API between you and AWS Batch
+to simplify interactions that otherwise involve repeatedly checking AWS SDK
+documentation.
+
+# How to read this introduction
+
+This introduction includes `scala` language code blocks and their output. The
+output it automatically generated by the [`mdoc`](https://scalameta.org/mdoc/)
+tool. The steps described in this introduction include sending JSON to HTTP
+endpoints. While the JSON in the examples is generated automatically from
+`scala` values, clients can produce JSON using whatever languages and tools
+they desire.
+
+# Background info
+
+To begin with, here are some IDs and imports that we'll be using throughout the examples.
+They are present for consistency and compilation throughout the rest of this
+document. You can ignore them.
+
+```scala mdoc
```

Comment:
Since this isn't used anywhere and I'm still worried that people are going to think they need to know `scala` to run `granary` - I think it might be better to mark this as [`invisible`](https://scalameta.org/mdoc/docs/modifiers.html#invisible)